README.md

```diff
@@ -1 +1,7 @@
-# Module1-Hello-Microverse
\ No newline at end of file
+This is my first microverse project into github.
+
+The content is a HTML and CSS file, two md files and a folder that contain linters file.
+
+Also in adition a file .gitignore that can hide a md file with the content "This file should be ignored by git" and the workflows folder with the linters.
+
+The HTML file only contain a title in h1 that say "Hello Microverse" and, the CSS file class set a font family style and color font.   
```

Comment:
- [ ] Kindly your readme from [here](https://github.com/microverseinc/readme-template) which is provided by microverse.